test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -833,6 +854,8 @@ get_mapping(Node, Client) ->
     {ok, What} = redis_query(Node, [<<"GET">>, FullJid]),
     What.
 
+%% Warning! May not work properly with alice or any other user whose
+%% stringprepped JID is different than original one
 delete_mapping(Node, Client) ->
```

Comment:
`escalus_utils:jid_to_lower(BinJid)` can be used.

Reply:
When I tried to experiment with this function, some tests suddenly broke. :P So I'm putting a warning for now so the PR won't be blocked... but this suite should be refactored definitely.

Reply:
yep, there is some regex inside, not sure it does normalization too